docs/DeveloperGuide.md

```diff
@@ -240,18 +240,18 @@ How the `Logic` component works:
 
 Figure 7 below illustrates the interactions within the `Logic` component for the `execute("delete-index 1")` API call.
 
-![Interactions Inside the Logic Component for the `delete 1` Command](images/DeleteSequenceDiagram.png)
+![Interactions Inside the Logic Component for the `delete 1` Command](images/DeleteIndexSequenceDiagram.png)
```

Comment:
This line should also change according to the command that you are trying to depict, as mentioned in line 241.